### ISSUE TITLE
Wunderlist cannot be contained

### ISSUE THREAD

**Issue by openjck** [mozilla/testpilot-containers#475]
### Steps to reproduce

1. Create two [Wunderlist](https://www.wunderlist.com/) accounts (we'll call them *A* and *B*)
    * Wunderlist allows you to choose a custom background image in settings, which can be used to differentiate them
2. Log out of both
3. In a non-container tab, log into account A
4. In any container tab, log into account B
5. Close both tabs
6. Quit Firefox
7. Re-open Firefox
8. In a non-container tab, visit [wunderlist.com](https://www.wunderlist.com/)

### Actual result

Account B is loaded

### Expected result

Account A should be loaded.

Why? Remember, once step 4 is completed the non-container tab is showing account A and the container tab is showing account B. Loading wunderlist.com from the non-contanier tab later on should show account A as it did before.

In other words, account B escapes its container. Not nice, account B!

**Comment by groovecoder** [comment]
Whoa ... this seems like a serious bug. @kjozwiak can you reproduce?

**Comment by jonathanKingston** [comment]
I think this is a dupe of this: https://github.com/mozilla/testpilot-containers/issues/397

**Comment by openjck** [comment]
I can't reproduce this any more. I think [bug 1334587](https://bugzilla.mozilla.org/show_bug.cgi?id=1334587) may have fixed it. Hooray! :tada:
